doc/api/crypto.md

```diff
@@ -5275,6 +5275,8 @@ would allow an attacker to guess one of the values. This is suitable for
 comparing HMAC digests or secret values like authentication cookies or
 [capability urls](https://www.w3.org/TR/capability-urls/).
 
+Throws an error if 'a' and 'b' are not equal length.
```

Comment:
This is not true. Their length may be different as long as their byte length is equal.

Also, this does not work well with the following paragraph, which _does_ say that the buffers must have the same byte length. Maybe the next paragraph could be modified to be more clear.

Reply:
This is the code path in the function that throws the error(I know because I used it today).

> if (buf1.size() != buf2.size()) {
    THROW_ERR_CRYPTO_TIMING_SAFE_EQUAL_LENGTH(env);
    return;
  }

Reply:
@alexagranov7 Yes, but that code path compares the `size` (which refers to the byte length), not the length.